apps/picsa-tools/crop-probability-tool/src/app/components/crop-probability-table-header/crop-probability-table-header.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'picsa-crop-probability-table-header',
+  templateUrl: './crop-probability-table-header.component.html',
+  styleUrls: ['./crop-probability-table-header.component.scss'],
+})
+export class CropProbabilityTableHeaderComponent {}
```

Comment:
nice idea to split this into it's own component, definitely helps keep the rest of the code a lot tidier